source/data/History.rst

```diff
@@ -76,3 +75,32 @@ The response will be:
       17.36
     ]
   }
+
+No data in requested period
+############################
+
+If there is no data in the requested period, consider the following:
+- Set the status code to ``no_data`` or return an empty response in case there is no historical data in the previous periods.
```

Comment:
List points are not on the separate lines.
![image](https://user-images.githubusercontent.com/22204671/207012920-2d2d74d5-355d-491e-b7ac-85bd6b85a2c2.png)